### DISCUSSION TITLE
LazyVim not updating in nvim

### DISCUSSION THREAD

**Discussion by crimpz** [LazyVim/LazyVim#624] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/18359937/233000720-7240abb5-9ccf-4af9-8750-b3659982f5b1.png)
Everything else updates but I keep getting this error when I try and update lazyvim, is it referencing the treesitter.lua in .local? If so, how do I fix this? Tried replacing it with the same one from the github repo but it didn't seem to fix anything.

**Comment by folke** [comment]
The error clearly shows you that you made changes to the LazyVim plugin files. You should never do that. Revert those changes to update.

You also seem to be having connectivity issues

**Reply by crimpz** [reply]
Yeah I changed one of the languages under ensure_installed. I tried undoing it and that didn't work, I also tried just changing it to the one from the repo and it still thinks I changed it. Is there a way I can stash or get rid of the changes it thinks I've done?

**Reply by folke** [reply, marked as answer]
go in that folder and do `git checkout -- .` This is really basic git usage though, so you'd better read some guides on how to work with git.

**Reply by crimpz** [reply]
Thanks for the help and quick responses! I'm still very new to nvim and github in general, so thank you.
